docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -118,13 +118,13 @@ But `async` and `await` are the real stars of this release. When these features
 
 With versions 3.0 and 5.0, C# had added major new features in an object-oriented language. With version 6.0, it would go away from doing a dominant killer feature and instead release many smaller features that made C# programming more productive. Here are some of them:
 
-- [Static imports](../language-reference/keywords/using-static.md)
-- [Exception filters](https://www.thomaslevesque.com/2015/06/21/exception-filters-in-c-6/)
-- [Property initializers](http://geekswithblogs.net/WinAZ/archive/2015/06/30/whatrsquos-new-in-c-6.0-auto-property-initializers.aspx)
-- [Expression bodied members](https://lostechies.com/jimmybogard/2015/12/17/c-6-feature-review-expression-bodied-function-members/)
-- [Null propagator](https://davefancher.com/2014/08/14/c-6-0-null-propagation-operator/)
-- [String interpolation](../language-reference/tokens/interpolated.md)
-- [nameof operator](../language-reference/keywords/nameof.md)
+- [Static imports](./csharp-6.md#using-static)
+- [Exception filters](./csharp-6.md#exception-filters)
+- [Auto-property initializers](./csharp-6.md#auto-property-initializers)
+- [Expression bodied members](./csharp-6.md#expression-bodied-function-members)
+- [Null propagator](./csharp-6.md#null-conditional-operators)
+- [String interpolation](./csharp-6.md#string-interpolation)
+- [nameof operator](./csharp-6.md#the-nameof-expression)
```

Comment:
@danielchalmers what has triggered you to switch the link from the language reference article to the section of the "What's new in.." article? I wonder why to make an additional stop-over at the "What's new..." article instead of short-circuiting to the language reference.

Reply:
just felt a bit strange having half the links go to the version history and half the links go to the reference.
IMO it would be better to just have a link going to the What's New pages, but that would be for another PR

Reply:
also the "What's New" pages give some nice context to the changes. For example:

> # out variables
> The existing syntax that supports out parameters has been improved in this version.
> 
> Previously, you would need to separate the declaration of the out variable and its initialization into two different statements:
>...